docs/configuring_tor.md

```diff
@@ -84,28 +79,41 @@ At this point, we can now start `lnd` with the relevant arguments:
 <snip>
 
 Tor:
-      --tor.socks=                             The port that Tor's exposed SOCKS5 proxy is listening on. Using Tor allows outbound-only connections (listening will be disabled) -- NOTE port must be between 1024 and 65535
-      --tor.dns=                               The DNS server as IP:PORT that Tor will use for SRV queries - NOTE must have TCP resolution enabled
+      --tor.active                                            Allow outbound and inbound connections to be routed through Tor
+      --tor.socks=                                            The port that Tor's exposed SOCKS5 proxy is listening on -- NOTE port must be between 1024 and 65535 (default: 9050)
+      --tor.dns=                                              The DNS server as IP:PORT that Tor will use for SRV queries - NOTE must have TCP resolution enabled (default: soa.nodes.lightning.directory:53)
+      --tor.streamisolation                                   Enable Tor stream isolation by randomizing user credentials for each connection.
```

Comment:
What does everyone think of defaulting to using streamisolation, and instead have a `no-streamisolation` option?